src/views/GithubCodePermalinkReferenceWidget.vue

```diff
@@ -134,7 +134,10 @@ export default {
 				: t('integration_github', 'Line {line}', { line: this.richObject.lineBegin })
 		},
 		shortRef() {
-			return this.richObject.ref.slice(0, 7)
+			if (this.richObject.ref.original_ref === this.richObject.ref.sha) {
```

Comment:
Is the sha already trimmed to 8 chars elsewhere?

Reply:
I don't know why I removed that. Anyway, the sha can always be shortened.